source/UF/CoconesofSpans.lagda

```diff
@@ -0,0 +1,503 @@
+Ian Ray, 15th January 2025
+
+We will prove some results about cocones of spans.
+
+\begin{code}
+
+{-# OPTIONS --safe --without-K #-}
+
+open import UF.FunExt
+
+module UF.CoconesofSpans (fe : Fun-Ext) where
+
+open import MLTT.Spartan
+open import UF.Base
+open import UF.Equiv
+open import UF.EquivalenceExamples
+open import UF.PropIndexedPiSigma
+open import UF.Retracts
+open import UF.Subsingletons
+open import UF.Yoneda
+
+\end{code}
+
+We start by defining cocones and characerizing their identity type.
+
+\begin{code}
+
+cocone : {A : 𝓤  ̇} {B : 𝓥  ̇} {C : 𝓦  ̇}
+         (f : C → A) (g : C → B) (X : 𝓣  ̇)
```

Comment:
The following should be module parameters. Have a look how things are done in `UF.Pullback`.

```
{A : 𝓤  ̇} {B : 𝓥  ̇} {C : 𝓦  ̇}
         (f : C → A) (g : C → B) (X : 𝓣  ̇)
```